econplayground/templates/assignment/assignment_questions.html

```diff
@@ -145,6 +145,11 @@ <h4>{{question.title}}</h4>
         selectQuestion(e.target, Number(e.target.value));
     });
 
+    $('select[name="graph"]').select2({
+        class: 'form-select',
```

Comment:
This selector should be changed to something more specific, to not accidentally get instantiated on other bootstrap select elements. Looks like we can use the `ep-question` class.